exec/api/src/main/java/org/jboss/windup/exec/configuration/WindupConfiguration.java

```diff
@@ -221,8 +230,7 @@ public WindupConfiguration addInputPath(Path inputPath)
      */
     public Collection<Path> getInputPaths()
     {
-        Collection<Path> inputPaths = getOptionValue(InputPathOption.NAME);
-        return inputPaths;
+        return getOptionValue(InputPathOption.NAME);
```

Comment:
Also, no need to change anything in this file since the original code and the new one are equivalent. Do you want to get the `getOptionValue(InputPathOption.NAME);` result for debugging purposes?

Reply:
This and other changes (the import ones) are changes I do as I develop the solution in order to cleanup the code and leave it in a slightly better shape that I found.